python/fledge/plugins/south/mqtt_sparkplug/sparkplug_b/sparkplug_b.proto

```diff
@@ -153,7 +153,7 @@ message Payload {
     optional MetaData metadata = 8;  // Metadata for the payload
     optional PropertySet properties = 9;
     oneof value {
-      uint32 int_value = 10;
+      int32 int_value = 10;
```

Comment:
Do not change auto generated file, this need not be changed ..

Reply:
This is necessary for handling signed integers. This generic MQTT SparkplugB publisher addresses the issue for any data type, eliminating the need for ctypes